docs/pages/careers/staff-engineer-pigment-css.md

```diff
@@ -0,0 +1,126 @@
+
+## The role
+
+### What you'll do on a day-to-day basis
+
+Depending on the day, you'll:
+
+- **Build**. You will provide solutions for integrating the library with various bundlers and frameworks. You will build new features and improve the DX of the library.
+- **Nurture community contributions**. You will provide guidance and direction to unlock the contributions of the community. Your time will often be way better spent doing this than fixing the problems yourself.
+- **Shape the product**. You will be laser-focused on the end goal. It's not about solving technical challenges but about the problem solved for the users.
+- **Keep technical debt in check**. You will make sure we can keep shipping features at a reasonable pace, align the team on "one way" of doing things and make sure engineers follow the conventions.
+- **Be a solver**. You will dig deep into arbitrarily complex problems and find an appropriate path forward. Some focus on a given area for long periods, others bounce from hotspot to hotspot as guided by organizational leadership.
+
+### The best parts of this job
+
+- **You'll be at the cutting edge of application development** — working on one of the fastest-growing CSS-in-JS libraries on the market.
+- **You'll be part of an active, open, friendly community** of developers that are excited about building awesome applications.
+- **Your role will be key to making MUI the go-to UI solution** for building applications, websites, and design systems that does not have to trade performance for a great developer expirience.
+
+### The worst parts of this job
+
+- **Shifting context.**
+  You will necessarily have to shift context and dive into a different feature before the current one is done.
+  It may even be in an area of the codebase you're unfamiliar with or don't have a ton of understanding about.
+  It's fun, rewarding work, but it can be very challenging.
+- **We move quickly but don't sacrifice quality**.
+  We ship early, often, and quickly. You may not be initially comfortable with the cadence with which we ship high-quality features and improvements to end-users. By doing so, we sacrifice on solving each problem 100% in exchange for fast feedback. Solving 50-70% of the issue with quality should be enough for any given iteration. Our users quickly tell us when we haven't pushed a solution far enough.
+- **MUI has a large codebase**. You may bang your head against the wall at times, and then write tests to assist future you 😌.
+  The work you will be doing is somewhat unique and idiosyncratic. You probably have not had a similar role before.
+- **In open-source, you're faced with a nonstop stream of bug reports and support requests**. That means you need to develop an intuition for when to ignore something, and when to dig in further.
+
+## Who we're looking for
```

Comment:
We can also be much simpler here! I removed the bold "titles" and just straight up added the required skills on Docs-infra role, as an example: https://mui.com/careers/react-engineer-docs-infra/#who-were-looking-for — that should be enough, I believe

Reply:
Should we aim in consistency of how we present the roles? We have template for starting, although we may change something in a particular job ad, all other roles would still be created differently.

For example, I changed the writing around the products in https://github.com/mui/material-ui/pull/42189/files, when it gets approved, I will apply this on the template. I think this is the only way to keep it consistent (if that's our goal).

Reply:
Yeah, definitely aim for consistency! We can keep updating the template as we see fit 👍

Reply:
I don't know. More details look OK. And since it's more verbose, we need a bold part for the quick job post readers. So 👍 to keep this as is on my end.